integrationtests/smb_test.go

```diff
@@ -129,37 +129,37 @@ func setupSmbShare(shareName, localPath, username string) error {
 		fmt.Sprintf("path=%s", localPath),
 		fmt.Sprintf("username=%s", username))
 	if output, err := cmd.CombinedOutput(); err != nil {
-		return fmt.Errorf("setupSmbShare failed: %v, output: %q", err, string(output))
+		return fmt.Errorf("setupSMBShare failed: %v, output: %q", err, string(output))
 	}
 
 	return nil
 }
 
-func removeSmbShare(t *testing.T, shareName string) {
+func removeSMBShare(t *testing.T, shareName string) {
 	cmdLine := fmt.Sprintf(`Remove-SMBShare -Name $Env:sharename -Force`)
 	cmd := exec.Command("powershell", "/c", cmdLine)
 	cmd.Env = append(os.Environ(),
 		fmt.Sprintf("sharename=%s", shareName))
 	if output, err := cmd.CombinedOutput(); err != nil {
-		t.Fatalf("setupSmbShare failed: %v, output: %q", err, string(output))
+		t.Fatalf("setupSMBShare failed: %v, output: %q", err, string(output))
 	}
 	return
 }
 
-func getSmbGlobalMapping(remotePath string) error {
+func getSMBGlobalMapping(remotePath string) error {
 	// use PowerShell Environment Variables to store user input string to prevent command line injection
 	// https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-5.1
-	cmdLine := fmt.Sprintf(`(Get-SmbGlobalMapping -RemotePath $Env:smbremotepath).Status`)
+	cmdLine := fmt.Sprintf(`(Get-SMBGlobalMapping -RemotePath $Env:smbremotepath).Status`)
```

Comment:
I'm not sure if this is case sensitive

Reply:
it's not https://learn.microsoft.com/en-us/powershell/module/smbshare/get-smbglobalmapping?view=windowsserver2022-ps, even if e2e succeeds I think we should use what the docs say

Reply:
Oops yeah that is totally fair. Went on auto pilot mode there.